react/typings/globals.d.ts

```diff
@@ -41,7 +43,7 @@ declare global {
   }
 
   interface MarketingData {
-    coupon: string
+    coupon: Maybe<string>
```

Comment:
it should be optional, no?
```suggestion
    coupon?: Maybe<string>
```

Reply:
it shouldn't matter much in this case, the issue lies when you want to cast an optional field into a required one, but it is fine the other way around